cmd/serve_test.go

```diff
@@ -206,6 +206,7 @@ func Test_serve(t *testing.T) {
 		Environment:                             "test",
 		EmailMessengerClient:                    messengerClient,
 		AdminDBConnectionPool:                   dbConnectionPool,
+		MtnDBConnectionPool:                     dbConnectionPool,
```

Comment:
💡 According to the Go naming conventions, acronyms should be either all-uppercased or all-lowercased. Plz, refer to the initialisms section from Go's official [CodeReviewComments wiki](https://go.dev/wiki/CodeReviewComments).

Reply:
Thanks, I was following the precedent set [here](https://github.com/stellar/stellar-disbursement-platform-backend/blob/develop/internal/serve/serve.go#L58), but I suppose we can fix that another time.